client/src/components/Dashboard/Board.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from "react";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+
+export default function Board(props) {
+  return (
+    <Fragment>
+      <Card
+        style={{ backgroundColor: props.backgroundColor }}
+        className={"board" + (props.className ? props.className : "")}
+        onClick={props.afterClick}
+      >
+        <CardContent>{props.children}</CardContent>
+        {props.showAction ? (
+          <CardActions style={{ justifyContent: "center" }}>
+            <Button size="small">View Board</Button>
+          </CardActions>
+        ) : (
+          ""
+        )}
```

Comment:
You directly use conditional rendering.

```js
{props.showAction && (
          <CardActions style={{ justifyContent: "center" }}>
            <Button size="small">View Board</Button>
          </CardActions>
        )}
```

Reply:
There are 3 - 4 types of conditional rendering, using this good but using conditional operator is also not bad.

Reply:
A ternary operator used for If-Else statements.
But this is only If statement.

Here, a good practice is to use a conditional operator.

...Because by using conditional operator I wouldn't have to think for else statement.